[EDU-6321] fix: adjust Account policies guide (#1547)
aziontech/docs: src/content/docs/pt-br/pages/guias/contas/conditional-access-by-ip.mdx

  "active": true,
  "rules": [
    {
      "name": "My policy rule",
      "effect": "allow",
      "resource": "platform",

>       "actions": "access",

Reviewer: ```suggestion
      "actions": ["access"],
```

}'
```

| Chave | Tipo | Descrição |
| -------- | ---------------- | --------------------------------------------------------------------------- |
| name | String | Refere-se ao nome da política de acesso condicional. Até *255 caracteres*.  |
| active | Boolean | Indica se a política está ativa. Valores aceitos: `true` e `false`. |
| rules | Array de objetos | Inclui um array de regras que definem o comportamento da política. |
| name | String | No array de regras, refere-se ao nome da regra individual. Até *255 caracteres*. |
| effect | String | Refere-se ao efeito da regra. Valores aceitos: `allow` e `deny`. |

> | resource | String | Especifica o recurso ou os recursos aos quais a regra se aplica, seja como uma URN exata ou um padrão de correspondência. **Notas**: este endpoint inclui o caso específico de `platform como recurso, usado exclusivamente para definir ações globais da plataforma. Valor aceito: .* (corresponde a todos os recursos). |

Reviewer 2: ```suggestion
| resource | String | Especifica o recurso ou os recursos aos quais a regra se aplica, seja como uma URN exata ou um padrão de correspondência. **Notas**: este endpoint inclui o caso específico de `platform` como recurso, usado exclusivamente para definir ações globais da plataforma. Valor aceito: `.*` (corresponde a todos os recursos). |
```

> | actions | String | Especifica a ação ou as ações às quais a regra se aplica. Valores aceitos: `create`, `retrieve`, `update` e `destroy`. **Nota**: ao usar `platform` como recurso, a única ação aceita é `access`. |

Reviewer: ```suggestion
| actions | Array de strings | Especifica a ação ou as ações às quais a regra se aplica. Valores aceitos: `create`, `retrieve`, `update` e `destroy`. **Nota**: ao usar `platform` como recurso, a única ação aceita é `access`. |
```

> | actions | Array de strings | Especifica a ação ou as ações às quais a regra se aplica. Valores aceitos: `create`, `retrieve`, `update` e `destroy`. **Nota**: ao usar `platform` como recurso, a única ação aceita é `access`. |

Reviewer 3: Palavras como `update` não estão em azul. Verificar.
